debian/changelog

```diff
@@ -39,7 +51,6 @@ daos (2.3.100-18) unstable; urgency=medium
   * Bump min supported go version to 1.16
 
  -- Michael MacDonald <mjmac.macdonald@intel.com> Tue, Jul 26 2022 10:10:00
--0400
```

Comment:
Looks like this wants to be joined to the line above it, not removed?

Reply:
doh, will fix